qubesbuilder/plugins/source_rpm/dnf/build-pkgs-base-fedora.list

```diff
@@ -0,0 +1,5 @@
+@buildsys-build
+@development-tools
```

Comment:
These are rather broad; I would prefer a list of the specific packages to be installed.

Reply:
From https://github.com/QubesOS/qubes-builder-rpm/blob/master/build-pkgs-base-fedora.list. We keep it sync for now.